synthetic_data/dataset_generators/categorical_generator.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
```

Comment:
let's move this into `distinct_generators` and move the previous.

Also, let's make sure the tests follow the same folder structure as well.

Reply:
The folder should be there now that #293 merged.